task/portal.go

```diff
@@ -71,7 +71,7 @@ func (p *Portal) Register(ctx context.Context, request *proto.RegisterRequest) (
 
 	return &proto.RegisterReply{
 		Addr:    "localhost:7100",
-		Subject: fmt.Sprintf("node-%s", request.Id),
+		Subject: "task",
```

Comment:
add a constant or method for "task" string?

Reply:
Maybe later, as these code should be refactor while the whole logic is stable.

Reply:
ok